pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -308,6 +324,32 @@ static void createTenantIfAbsent(ZooKeeper configStoreZk, String tenant, String
         }
     }
 
+    static void createTenantIfAbsent(MetadataStore configStore, String tenant, String cluster) throws IOException,
```

Comment:
Does this method https://github.com/apache/pulsar/pull/10600/files#diff-03a1201aab4aa284723e575a7c9b18ad9542fcc02beb80d83cfb87d64adf54b5R300 can be removed?

Reply:
As @BewareMyPower pointed out, these methods are still used in `PulsarInitialNamespaceSetup` and `PulsarTransactionCoordinatorMetadataSetup`. I plan to open more PRs to fix these commands as well and then remove these methods altogether.

Reply:
I see, thanks.